channels/us.m3u

```diff
@@ -424,139 +424,139 @@ https://bloodydisgusting-roku-us.cinedigm.com/midroll/amagi_hls_data_xumo-host-r
 https://bloomberg-bloombergtv-1-es.samsung.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="BloombergTVPlusUS.us" tvg-name="Bloomberg TV+ US" tvg-country="US" tvg-language="English" tvg-logo="https://1.bp.blogspot.com/-PbJbgwulmgg/XlcvWvE1krI/AAAAAAAArqk/GybN3PTg-tAskuSoJQe3hfT2CvPXdA1WwCLcBGAsYHQ/s320/Bloomberg%2BTelevision.jpg" group-title="Business",Bloomberg TV+ US (1080p)
+#EXTINF:-1 tvg-id="BloombergTVPlusUS.us" tvg-name="Bloomberg TV+ US" tvg-country="US" tvg-language="English" tvg-logo="http://cdn-profiles.tunein.com/s47135/images/logog.png" group-title="Business",Bloomberg TV+ US (1080p)
 https://liveprodusphoenixeast.akamaized.net/USPhx-HD/Channel-TX-USPhx-AWS-virginia-1/Source-USPhx-16k-1-s6lk2-BP-07-02-81ykIWnsMsg_live.m3u8
-#EXTINF:-1 tvg-id="BloombergUSDesktop.us" tvg-name="Bloomberg US Desktop" tvg-country="US" tvg-language="" tvg-logo="https://raw.githubusercontent.com/geonsey/Free2ViewTV/master/images/logos/Bloomberg_512x512.png" group-title="Business",Bloomberg US Desktop
+#EXTINF:-1 tvg-id="BloombergUSDesktop.us" tvg-name="Bloomberg US Desktop" tvg-country="US" tvg-language="" tvg-logo="http://cdn-profiles.tunein.com/s47135/images/logog.png" group-title="Business",Bloomberg US Desktop
 https://liveprodusphoenixeast.global.ssl.fastly.net/USPhx-HD/Channel-TX-USPhx-AWS-virginia-1/Source-USPhx-16k-1-s6lk2-BP-07-03-0Yn1cQZHOtP_live.m3u8
-#EXTINF:-1 tvg-id="BloombergMeiGuo.us" tvg-name="Bloomberg(美国)" tvg-country="US" tvg-language="English" tvg-logo="http://static.epg.best/us/BloombergTV.us.png" group-title="Business",Bloomberg(美国)
+#EXTINF:-1 tvg-id="BloombergMeiGuo.us" tvg-name="Bloomberg(美国)" tvg-country="US" tvg-language="English" tvg-logo="http://cdn-profiles.tunein.com/s47135/images/logog.png" group-title="Business",Bloomberg(美国)
 https://liveproduseast.global.ssl.fastly.net/us/Channel-USTV-AWS-virginia-1/Source-USTV-1000-1_live.m3u8
-#EXTINF:-1 tvg-id="BloombergPlus4K.us" tvg-name="Bloomberg+ 4K" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/idRFfhY.png" group-title="Business",Bloomberg+ 4K (2160p)
+#EXTINF:-1 tvg-id="BloombergPlus4K.us" tvg-name="Bloomberg+ 4K" tvg-country="US" tvg-language="English" tvg-logo="http://cdn-profiles.tunein.com/s47135/images/logog.png" group-title="Business",Bloomberg+ 4K (2160p)
 https://bloomberg-bloombergtv-1-gb.samsung.wurl.com/manifest/playlist.m3u8
 #EXTINF:-1 tvg-id="Bluegrass4U.us" tvg-name="Bluegrass 4U" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/nm3N3aw.png" group-title="",Bluegrass 4U
 https://59d39900ebfb8.streamlock.net/blugrassmusic/blugrassmusic/playlist.m3u8
-#EXTINF:-1 tvg-id="BonAppetit.us" tvg-name="Bon Appetit" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/YhFFxlE.png" group-title="",Bon Appetit
+#EXTINF:-1 tvg-id="bonappétit.us" tvg-name="bon appétit" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/YhFFxlE.png" group-title="",bon appétit
```

Comment:
Inconsistency with the tags below

Reply:
This is what I have: #EXTINF:-1 tvg-id="bonappétit.us" tvg-name="bon appétit"

I have been have problems with my commit push through. That's how their channel logo looks

Reply:
I thought the tvg-id could have spaces

Reply:
TVG-id doesn't have spaces indeed. said this because the end of that line has got a different name that the one you gave below.

Reply:
I have four of the channels, and they all look the same except the (1080p). That's allowed right?

Reply:
That's OK.